akka-stream/src/main/boilerplate/akka/stream/javadsl/SourceCreate.scala.template

```diff
@@ -9,13 +9,34 @@ import akka.stream.scaladsl.JavaConverters._
 
 trait SourceCreate {
 
+  /**
+   * Creates a `Source` by using a `FlowGraph.Builder[Unit]` on a block that expects
+   * a [[FlowGraph.Builder]] and returns an [[Outlet]].
+   */
   def create[T](block: japi.Function[FlowGraph.Builder[Unit], Outlet[T]]): Source[T, Unit] =
     new Source(scaladsl.Source() { b ⇒ block.apply(b.asJava) })
 
-  def create[T, S <: Shape, M](g1: Graph[S, M], block: japi.Function2[FlowGraph.Builder[M], S, Outlet[T]]): Source[T, M] =
+  /**
+   * Creates a `Source` by using a `FlowGraph.Builder[M]` on a block that expects
+   * a [[FlowGraph.Builder]] and 1 graph and then returns an [[Outlet]].
```

Comment:
perhaps mention that the given graphs are imported as with `Builder.graph` and the resulting shapes are passed to the block